static/app/components/modals/widgetViewerModal.spec.tsx

```diff
@@ -1393,6 +1487,20 @@ describe('Modals -> WidgetViewerModal', function () {
       );
       // Need to manually set the new router location and rerender to simulate the sortable column click
       initialData.router.location.query = {sort: '-sum(session)'};
+      const dashboardLegendUtils = new DashboardLegendEncoderDecoder({
+        location: initialData.router.location,
+        dashboard: {
+          id: 'new',
+          title: 'Dashboard',
+          createdBy: undefined,
+          dateCreated: '2020-01-01T00:00:00.000Z',
+          widgets: [mockWidget],
+          projects: [],
+          filters: {},
+        },
+        organization: initialData.organization,
+        router: initialData.router,
+      });
```

Comment:
Could you save this in a variable at the top of the spec and re-use for all tests?